### PR TITLE
Fix action item deprecation warnings

### PR DESCRIPTION
**Pull request by 1000miles**
This PR fixes the deprecation warnings on `action_item`. **Please merge #278 first.**
Related Issue: #236

### CODE REVIEW

File: app/admin/provider.rb [hunk 1]
```diff
@@ -4,10 +4,12 @@
...
-  action_item do
-    link_to upload_csv_admin_provided_pois_path do
-      "Upload CSV"
-    end
+  action_item(:upload_csv, :only => :index) do
+    link_to "Upload CSV", upload_csv_admin_provided_pois_path
+  end
+
+  collection_action :upload_csv, title: "Upload CSV" do
+    render "/admin/provided_pois/upload_csv"
```
Comment (schultyy) [inline app/admin/provider.rb]: What does `collection_action` do here?

**Comment by schultyy** [comment]
Why are also changes in this PR which are not related to ActiveAdmin `action_item`?

**Comment by 1000miles** [comment]
This was a separate test on `provided_poi`. Thanks for the hint. I will remove the `collection_action` part.

**Comment by schultyy** [comment]
There are still changes which do not belong to ActiveAdmin:
- `app/models/poi.rb`
- `app/models/provided_poi.rb`
- `app/models/user.rb`

**Pull request merged**